network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -198,6 +200,12 @@ public static String buildCloneIdentifiablesQuery(String tableName, Collection<S
                 "where networkUuid = ? and variantNum = ?";
     }
 
+    public static String buildCloneTemporaryLimitsQuery() {
+        return "insert into temporarylimit(equipmentId, equipmentType, networkUuid, variantNum, side, limitType, name, value, acceptableDuration, fictitious) " +
```

Comment:
maybe you can use upper case for SQL keywords like for the other queries

Reply:
This is already the case. Everything is in lowercase in this file.